.github/workflows/test-workflow.yml

```diff
@@ -20,6 +20,14 @@ jobs:
       with:
         node-version: "${{ steps.nvm.outputs.NVMRC }}"
 
+    - name: Cache Dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
```

Comment:
@MattIPv4 No, it is not needed! We are caching it in the system level!

And official document says:

> Note: It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci

https://github.com/actions/cache/blob/master/examples.md#node---npm

Reply:
Ah, so this is just caching the registry data for the packages, instead of caching the actual deps?

Reply:
Yes!